migration/v0.42/apply/10-upgrade-grafana.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+ROOT="$(readlink -f "$(dirname "${0}")/../../../")"
+
+# shellcheck source=scripts/migration/lib.sh
+source "${ROOT}/scripts/migration/lib.sh"
+
+run() {
+  case "${1:-}" in
+  execute)
+    if [[ "${CK8S_CLUSTER}" =~ ^(sc|both)$ ]]; then
+      log_info "Deleting existing Grafana releases to address volumeName templating issue"
+
+      mapfile -t helm_secrets < <(kubectl_do sc get secrets -n monitoring -oname | grep -E 'sh.helm.release.v1.ops|sh.helm.release.v1.user-grafana')
```

Comment:
Feels like it would be easier if you did a get and then delete by labels.

Reply:
They don't have any common labels that target only those secrets and nothing else, so that would have to be two separate kubectl commands for user- and ops-grafana without doing any piping shenanigans since kubectl does not seem to support OR:ing label selectors. Is that still preferable?

Reply:
Yes, I'd say so.